Misc/NEWS.d/next/Library/2019-05-09-22-36-41.bpo-33303.B7_POx.rst

```diff
@@ -0,0 +1 @@
+properly escape XML comment text
```

Comment:
This is meant for humans to read, so please write a complete sentence, with a dot at the end.

Reply:
Done